Control/DeepSeq.hs

```diff
@@ -491,10 +623,18 @@ instance NFData Fingerprint where
 -- Foreign.Ptr
 
 -- |@since 1.4.2.0
-instance NFData (Ptr a) where rnf = rwhnf
+instance NFData (Ptr a) where
+    rnf = rwhnf
+-- |@since 1.4.3.0
+instance NFData1 MVar where
```

Comment:
I think if you changed this to `instance NFData1 Ptr`, the Travis build would work again ;)